pom.xml

```diff
@@ -91,7 +91,7 @@
         <dependency>
             <groupId>org.jboss.marshalling</groupId>
             <artifactId>jboss-marshalling-river</artifactId>
-            <version>1.4.9.Final</version> <!-- TODO JENKINS-32986 need 1.4.12.Final when released to pick up JBMAR-189 https://github.com/jboss-remoting/jboss-marshalling/pull/48 -->
+            <version>1.4.12.jenkins-3</version> <!-- https://github.com/jglick/jboss-marshalling/compare/1.4.12.Final...1.4.12.jenkins-3 -->
```

Comment:
[Custom patches](https://github.com/jglick/jboss-marshalling/compare/1.4.12.Final...1.4.12.jenkins-3) and [aggregate changes](https://github.com/jglick/jboss-marshalling/compare/1.4.9.Final...1.4.12.jenkins-3)